uni/lib/model/entities/login_exceptions.dart

```diff
@@ -1,13 +1,16 @@
+import 'package:uni/generated/l10n.dart';
+import 'package:uni/main.dart';
+
 class ExpiredCredentialsException implements Exception {
   ExpiredCredentialsException();
 }
 
 class InternetStatusException implements Exception {
   InternetStatusException();
-  String message = 'Verifica a tua ligação à internet';
+  String message = S.of(MyApp.navigatorKey.currentContext!).check_internet;
 }
 
 class WrongCredentialsException implements Exception {
   WrongCredentialsException();
-  String message = 'Credenciais inválidas';
+  String message = S.of(MyApp.navigatorKey.currentContext!).invalid_credentials;
 }
```

Comment:
While this works due to not being tested, it should receive an AppLocale the same way the Exam does to keep clear separation between the models and the view. (I believe it should be fine to remove that context trick everywhere now)

Reply:
How can I get access to `context` in session_provider.dart, so that I can get the current locale?

Reply:
You don't need to, just have it receive the Locale since it needs to display stuff

Reply:
In order to pass the Locale, we need to call the provider with context